### DISCUSSION TITLE
Error: Something went wrong

### DISCUSSION THREAD

**Discussion by acedweb** [AntlerVC/firetable#928] [Support / Q&A] [answered]
i deleted the project and try to recreate and connect to an existing project then i get the error Something went wrong

**Comment by BeeBombshell** [comment]
Hey @acedweb, If you could elaborate on the problem you are having or provide any screenshots, that would be helpful. You can also reach out to us directly at [support@rowy.io](mailto:support@rowy.io) to help you resolve the issue you're facing.

**Comment by brndn-pex** [comment]
I have the same issue, something with the identitykit api

**Comment by BeeBombshell** [comment, marked as answer]
Hey @acedweb and @brndn-pex, is this still an issue?

**Reply by BeeBombshell** [reply]
Hey guys, closing this discussion since it has been unattended for a long time. @acedweb, @brndn-pex, if you're still facing this issue, feel free to create a new one.

**Comment by bofeiw** [comment]
We added a self-service "disconnect" button, to connect an existing project, you can disconnect it during project setup.


![image](https://github.com/rowyio/rowy/assets/34177142/bc5eba74-2ad6-4e88-a43c-019b5764f354)
